services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/PublicHankeController.kt

```diff
@@ -40,5 +40,7 @@ class PublicHankeController(private val hankeService: HankeService) {
                 )
             ]
     )
-    fun getAll(): List<PublicHanke> = hankeService.loadPublicHanke().map { hankeToPublic(it) }
+    fun getAll(): List<PublicHanke> {
```

Comment:
Why the change? It's the same thing but with three lines.

Reply:
I used the endpoint for some early prototyping and didn't notice I left a change. Reverted this. Thank you for noticing.